subject_files_status_for_dpdash2.py

```diff
@@ -28,8 +28,22 @@ def get_value(event,var):
     return ''
 
 
+def get_value_from_dict(dict1, event, var):
+    """Extract value from JSON"""
 
-def get_mri_status():
+    for d in dict1:
+        if event in d['redcap_event_name']:
+            try:
+                if d[var] != '':
+                    return d[var]
+            except KeyError:
+                pass
+
+    # the subject has not reached the event yet
+    return ''
+
+
+def get_mri_status(dict1, timepoint, consent_date, subject, test=False):
```

Comment:
Again, Tashrif has designed the program in a way that `dict1, timepoint, consent_date, subject` are already available within the function. `test=False`, although a new parameter, is also covered by his design.

Reply:
[Same answer](https://github.com/AMP-SCZ/utility/pull/106#discussion_r1496209809)